### PR TITLE
Fix Faure sequences

### PR DESCRIPTION
**Pull request by ParadaCarleton**
Removes randomization from Faure sequences after discovering a bug, and updates tests to make sure a mistake doesn't happen again.

@dmetivie has done an amazing job implementing randomization for arbitrary bases in his RandomizedQuasiMonteCarlo.jl package!

### CODE REVIEW

**Comment by ParadaCarleton** [comment]
Someday I'll figure out how to create a pull request without creating a huge mess of commit messages, but today is apparently not that day

File: test/Manifest.toml [hunk 1]
```diff
@@ -0,0 +1,468 @@
+# This file is machine-generated - editing it directly is not advised
+
+julia_version = "1.8.2"
+manifest_format = "2.0"
+project_hash = "23f0ca1a3f884f478a58053e947c897723657745"
```
Comment (ChrisRackauckas) [inline test/Manifest.toml:5]: there should not be a manifest in the tests

Comment (ParadaCarleton) [reply test/Manifest.toml]: Whoops, thank you.

File: src/Faure.jl [hunk 2]
```diff
@@ -40,11 +40,7 @@ References:
 Faure, H. (1982). Discrepance de suites associees a un systeme de numeration (en dimension s). *Acta Arith.*, 41, 337-351.
 Owen, A. B. (1997). Monte Carlo variance of scrambled net quadrature. *SIAM Journal on Numerical Analysis*, 34(5), 1884-1910.
 """
-struct FaureSample{T}
-    rng::T
-end
-
-FaureSample() = FaureSample(Random.GLOBAL_RNG)
+struct FaureSample end
```
[the remark this answers is not in the document]
Comment (ParadaCarleton) [reply src/Faure.jl:43]: As I mentioned at the top of the thread, the current implementation contains some bugs. @dmetivie has provided an excellent implementation for Owen scrambling, which we'll hopefully be able to add directly to QMC.jl in a pull request.

**Pull request merged**